### PR TITLE
Shivangi_Assignment13_indexOf_And_lastIndexOf_Examples

### PR DESCRIPTION
**Pull request by ShivangiTechnocredits**
AllDuplicateChar -
image

allUniqueChar -
image

firtUniqueChar -
image

firstDuplicate -
image

### CODE REVIEW

File: src/shivangi/A13_indexOf.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package shivangi;
+
+public class A13_indexOf {
+	void AllDuplicateChar(String name){
```
Comment (KrishnaKTechnocredits) [inline src/shivangi/A13_indexOf.java:4]: method name should starts with small letter

**Review by KrishnaKTechnocredits — APPROVED** [review]
The code logic is good to me! keep up the good work @ShivangiTechnocredits

File: src/shivangi/A13_indexOf.java (hunk 1 above)
Comment (ShivangiTechnocredits) [reply src/shivangi/A13_indexOf.java:4]: Oh sorry @KrishnaKTechnocredits mam it happened by mistake, other than are okay mam?

**Comment by ShivangiTechnocredits** [comment]
Okay @KrishnaKTechnocredits mam Thank you!!

Will try my best.

**Pull request merged**